src/components/settings/ContractVersion/index.tsx

```diff
@@ -42,6 +44,24 @@ export const ContractVersion = ({ isGranted }: { isGranted: boolean }) => {
           Unsupported contract
         </Typography>
       )}
+      <div style={{ marginTop: 15 }}>
+        {needsUpdate ? (
+          <Tooltip
+            title="Update now to take advantage of new features and the highest security standards available.
+You will need to confirm this update just like any other transaction."
+            placement="right-start"
+          >
+            <Typography variant="body2" style={{ display: 'flex', alignItems: 'center' }}>
```

Comment:
I suggest we keep the text the same size as the other message and use Emption props instead:

```suggestion
            <Typography display="flex" alignItems="center">
```